variables.tf

```diff
@@ -27,7 +27,7 @@ variable "cors_configuration" {
     allow_headers     = optional(list(string))
     allow_methods     = optional(list(string))
     allow_origins     = optional(list(string))
-    expose_headers    = optional(list(string), [])
```

Comment:
I think this might be a valid change - will test it out

Reply:
not valid - updated in https://github.com/terraform-aws-modules/terraform-aws-apigateway-v2/pull/114/commits/b7ec703569f213a510721a884430865e59550cab